charforge-gui/start-dev.sh

```
@@ -0,0 +1,139 @@
+fi
+
+if ! command_exists npm; then
+    echo "❌ npm is required but not installed"
+    exit 1
+fi
+
+echo "✅ Dependencies check passed"
+
+# Setup backend
+echo "🔧 Setting up backend..."
+cd backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)